html_css/intermediate_html/form-validations.md

```diff
@@ -197,6 +205,7 @@ It's also worth noting client-side validations are not a silver bullet for ensur
 - Check out [html5pattern](https://www.html5pattern.com/) for a list of commonly used pattern regular expressions you may find helpful.
 - Look through [this Twitter thread](https://twitter.com/vponamariov/status/1400388896136040454) of the do and don'ts for form validation UX.
 - [10 Guidelines](https://www.nngroup.com/articles/errors-forms-design-guidelines/) for form validation design
+- Have a look through [regular expressions](https://towardsdatascience.com/regular-expressions-clearly-explained-with-examples-822d76b037b4) clearly explained in this article.
```

Comment:
After looking over this link, I think it could be beneficial for users who wish to look into it to see a regex pattern and have each piece walked through. That said, I think we should also include an MDN article to further explain things. Let's change this to:

```suggestion
- [Regular expressions clearly explained](https://towardsdatascience.com/regular-expressions-clearly-explained-with-examples-822d76b037b4) goes over several real world examples of regular expression patterns and walks through what they're doing step by step. This resource can be useful if you don't just want to copy + paste popular patterns, and instead want to know what a pattern is actually doing.
- One last resource on regular expressions that can be helpful is MDN's [regular expression syntax cheatsheet](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Cheatsheet). This cheatsheet does a great job explaining the syntax of regular expressions in more detail.
```